include/swift/Option/Options.td

```diff
@@ -148,6 +148,10 @@ def F : JoinedOrSeparate<["-"], "F">, Flags<[FrontendOption]>,
   HelpText<"Add directory to framework search path">;
 def F_EQ : Joined<["-"], "F=">, Flags<[FrontendOption]>, Alias<F>;
 
+def Fsystem : JoinedOrSeparate<["-"], "Fsystem">, Flags<[FrontendOption]>,
```

Comment:
This is about the *non* '=' version. You can say `-F/foo/bar/`, but I think `-Fsystem/foo/bar/` is probably a bad idea.

(I'm okay with not adding a `-Fsystem=` option too.)